cmd/kubeadm/app/util/logfile.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+// kubeadm record file path and name define
+const (
+	KubeadmLogFolder = "/var/lib/kubeadm"
+	KubeadmLogFile   = KubeadmLogFolder + "/" + "kubeadm.log"
```

Comment:
- let's lowercase these 2 constants.
because we don't need them in other packages for now.
- then later use filepath.Join() to join folders and files this is to be able to make the implementation portable on Windows.

also add constants for the tests (will provide more details later):
```
kubeadmLogFolder     = "/var/lib/kubeadm"
kubeadmLogFile       = "kubeadm.log"
kubeadmLogTestFolder = os.TempDir()
kubeadmLogTestFile   = "kubeadm-log-file-test"
```